driver/InputFlags.cpp

```diff
@@ -102,6 +105,7 @@ char InputFlags::FindShortName(const std::string& long_name) const
 
 void InputFlags::Parse(int argc, char* argv[])
 {
+
```

Comment:
Empty line is for readability or accidentally added?

Reply:
removed the empty line , as it was extra.

Reply:
@technicalgrp89 your latest commit not yet pushed?

Reply:
@junliume changes have been pushed & submitted for review.